File: Ejercicios PHP/10.Ley Ohm.php
<?php
echo('Ingrese el valor del voltaje del circuito: ');
$Voltaje = (int)(fgets(STDIN));
echo('Ingrese el valor de la resistencia del circuito: ');
$Resistencia = (int)(fgets(STDIN));
$Intensidad = $Voltaje/$Resistencia; //Intesidad entiendase por amperaje
echo("Al conectar un resistor de: $Resistencia ohm, a una fuente de: $Voltaje V, El voltage circulara una corriente de $Intensidad amperios");
//Para ejecutar en terminal: php "C:\Sena\Analisis y desarrollo de software\Ficha 3066478 segundo trimestre\Ejercicios PHP\10.Ley Ohm.php"
?>

File: Ejercicios GO/10.Ley Ohm.go
package main

import "fmt"

func main(){
	var Voltaje, Resistencia, Intensidad float64
	fmt.Println("Ingrese el valor del voltaje del circuito: ")
	fmt.Scanln(Voltaje)
	fmt.Println("Ingrese el valor de la resistencia del circuito: ")
	fmt.Scanln(Resistencia)
	Intensidad = (Voltaje/Resistencia)
	fmt.Printf("Al conectar un resistor de %.2f ohm, a una fuente de %.2f V el voltage circulara una corriente de %.2f amperios", Voltaje, Resistencia, Intensidad)
}	
